server/tests/unit/api/room.spec.ts

```diff
@@ -225,6 +225,13 @@ describe("Room API", () => {
 					isTemporary: true,
 				},
 			],
+			[
+				{ arg: "title", reason: "not allowed (too long, must be at most 255 characters)" },
+				{
+					name: "abababababababababababababababababababababababababababababababababababababababababababababababababababababababababababababababababababababababababababababababababababababababababababababababababababababababababababababababababababababababababababababababab",
```

Comment:
The field you should be testing for is `title`, not `name`. It's still required to provide a `name` when creating a room, so set it to something valid like `foo`.

Room name is different from the room title. Room name is used like a unique identifier for the room, and appears in URLs.

Reply:
Made those changes. Should be good now I think.